matrixctl/sanitizers.py

```diff
@@ -120,6 +120,21 @@ def sanitize_message_type(
     return False
 
 
+def sanitize(
```

Comment:
The code looks good to me. If you add a [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html)-style docstring to the `sanitize` function, everything should be good to go. You can also check it, and run tests by running `poetry run tox`.